clients/fossid-webapp/src/test/kotlin/FossIdClientNewProjectTest.kt

```diff
@@ -113,6 +113,7 @@ class FossIdClientNewProjectTest : StringSpec({
         service.createScan(
             "", "",
             PROJECT_CODE,
+            SCAN_CODE,
```

Comment:
I general, all modification to files in the `clients` directory should IMO be split out to a preparing commit.